src/modules/categories/components/CategoryList.js

```diff
@@ -18,17 +20,27 @@ type PropsType = {|
   query?: string,
   theme: ThemeType,
   onItemPress: (tile: { title: string, thumbnail: string, path: string }) => void,
-  language: string
+  language: string,
+  thumbnail: string
```

Comment:
You should use `thumbnail?: string`, so you don't have to pass anything if no thumbnail should be rendered.

Reply:
Or `thumbnail: ?string`